domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -289,7 +301,7 @@ constructor(
             } ?: false
 
         val previewUseCaseBuilder = Preview.Builder()
-        if (isPreviewStabilizationSupported) {
+        if (isPreviewStabilizationSupported && shouldStabilize) {
```

Comment:
Do we also have to set `setPreviewStabilizationEnabled` to `false` if we don't meet the criteria?

Reply:
I intentionally chose to leave it alone as unspecified, since we aren't explicitly saying to leave it off. setPreviewStabilizationEnabled can be set to true (on), false (off), or leaving it alone to be unspecified, since setting preview or video stabilization may be different. 

Noting that, something I still need to test is if the fps selection issue still occurs when *only* video capture is stabilized.

<img width="474" alt="image" src="https://github.com/google/jetpack-camera-app/assets/20013168/32cc5584-4b45-4b55-ad00-c4fb26203216">